index.bs

```diff
@@ -369,71 +366,11 @@ QUIC connection.
 
 The [=agent certificate=] must have the following characteristics:
 
-* 256-bit or 384-bit ECDSA public key.
+* 256-bit ECDSA public key.
 * Self-signed.
-* Supporting at least one of the [=certificate algorithms=] listed below.
-    * The `AlgorithmIdentifiers` are defined in [[!RFC5480]] (for public keys) and
-        [[!RFC5758]] (for signature schemes).
-    * [[!X690]] specifies the Distinguished Encoding Rules (DER) representation used to encode the identifiers.
+* Support the `secp256r1_sha256` signature algorithm.
```

Comment:
Is this defined somewhere in this document? I couldn't find it if so.

Would love a link to the spec for this. It's also the same thing as prime256v1 right?